extensions/amp-story-interactive/0.1/test/test-amp-story-interactive-img-poll.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AmpStoryInteractiveImgPoll} from '../amp-story-interactive-img-poll';
+import {AmpStoryRequestService} from '../../../amp-story/1.0/amp-story-request-service';
+import {AmpStoryStoreService} from '../../../amp-story/1.0/amp-story-store-service';
+import {LocalizationService} from '#service/localization';
+import {Services} from '#service';
+import {addConfigToInteractive} from './test-amp-story-interactive';
+import {measureMutateElementStub} from '#testing/test-helper';
+import {registerServiceBuilder} from '../../../../src/service-helpers';
+
+describes.realWin(
+  'amp-story-interactive-img-poll',
+  {
+    amp: true,
+  },
+  (env) => {
+    let win;
+    let ampStoryPoll;
+    let storyEl;
+    let requestService;
+
+    beforeEach(() => {
+      win = env.win;
+
+      env.sandbox
+        .stub(Services, 'cidForDoc')
+        .resolves({get: () => Promise.resolve('cid')});
+
+      const ampStoryPollEl = win.document.createElement(
+        'amp-story-interactive-img-poll'
+      );
+      ampStoryPollEl.getResources = () => win.__AMP_SERVICES.resources.obj;
+      requestService = new AmpStoryRequestService(win);
+      registerServiceBuilder(win, 'story-request', function () {
+        return requestService;
+      });
+
+      const storeService = new AmpStoryStoreService(win);
+      registerServiceBuilder(win, 'story-store', function () {
+        return storeService;
+      });
+
+      const localizationService = new LocalizationService(win.document.body);
+      env.sandbox
+        .stub(Services, 'localizationServiceForOrNull')
+        .returns(Promise.resolve(localizationService));
```

Comment:
I think you don't need any of this setup for the tests you wrote. Try removing these lines and check if the tests pass